api/views.py

```diff
@@ -1,3 +1,56 @@
-from django.shortcuts import render
+from django.http import JsonResponse
+from rest_framework_simplejwt.tokens import RefreshToken
 
+from .serializers import *
+from .models import *
+from rest_framework import viewsets, views
 # Create your views here.
+
+
+class UserViewSet(viewsets.ModelViewSet):
+    serializer_class = UserSerializer
+    queryset = User.objects.all()
+
+
+class SignUpView(views.APIView):
+    serializer_class = SignUpSerializer
+
+    def post(self, request):
+        serializer = self.serializer_class(data=request.data)
+
+        if serializer.is_valid():
+            user = serializer.save()
+
+            token = RefreshToken.for_user(user)
+            refresh = str(token)
+            access = str(token.access_token)
+
+            return JsonResponse({
+                'user': user.username,
+                'refresh': refresh,
+                'access': access,
+            })
```

Comment:
```suggestion
            return JsonResponse({
                'message': '회원가입 성공',
                'user': user.username,
                'access': access,
                'refresh': refresh,
            }, status = 201)
```

이런 방식의 응답은 어떨지 제안해봅니다..! 메세지와 상태코드 명시해주면 좋을 것 같고, access와 refresh의 순으로 리턴해주는 게 깔끔하지 않을까 싶어요 ☺️

상태코드도 DRF의 status 모듈에서 HTTP_201_CREATED 가져와주면 더더욱 클린한 코드가 되지 않을까 생각합니다~ 상태코드 부분은 제가 작업하면서 추가해놓을게용